### PR TITLE
[skip ci] feat: added section about http interception in CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by alekc**
Checklist:

- [ ] Updated `extVersionCode` value in `build.gradle` for individual extensions
- [ ] Updated `overrideVersionCode` or `baseVersionCode` as needed for all multisrc extensions
- [ ] Referenced all related issues in the PR body (e.g. "Closes #xyz")
- [ ] Added the `isNsfw = true` flag in `build.gradle` when appropriate
- [ ] Have not changed source names
- [ ] Have explicitly kept the `id` if a source's name or language were changed
- [ ] Have tested the modifications by compiling and running the extension through Android Studio

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -429,6 +429,56 @@ Instead, once you've built and installed your extension on the target device, us
...
+and point your browser to http://0.0.0.0:8081/
+
+#### OkHttp proxy setup
+Since most of the manga sources are going to use https we need to disable ssl verification in order to use web debugger.
+
+Amend your source kotlin file (here we are using MangaBuddy)
+```kotlin
+class MangaBuddy : MadTheme(
+    "MangaBuddy",
+    "https://mangabuddy.com",
```
Comment (CarlosEsco) [inline CONTRIBUTING.md:452]: Just make this reference a random non real source / theme.  and its good to go.

Comment (alekc) [reply CONTRIBUTING.md]: done

**Comment by alessandrojean** [comment]
Why not just use Logcat and filter for `OkHttpClient` tag instead? The app already has a logger inside it if you use the Preview version (as intended for developing extensions).

Using Logcat doesn't need any SSL bypass and other workarounds done that mitm requires.

**Comment by alekc** [comment]
Well, first of all, I was not aware of that possibility :)

As for the rest: I've tried to follow your instruction, but I still can't see the HTTP flow from the logcat.

Also, I'd rather prefer having webui which I can easily click through/expand as I need rather then parsing logcat output.

**Pull request merged**